daktari/checks/misc.py

```diff
@@ -139,6 +139,7 @@ class ShfmtInstalled(Check):
 
     suggestions = {
         OS.OS_X: "<cmd>brew install shfmt</cmd>",
+        OS.UBUNTU: "<cmd>snap install shfmt</cmd>",
```

Comment:
https://snapcraft.io/install/shfmt/ubuntu it exists 😄 on their instructions it says to `sudo` it but I guess that doesn't matter?

Reply:
yeah, for me at least running without sudo popped up a pw prompt, which was fine :shrug: